addons/rose/addon/helpers/set-from-event.js

```diff
@@ -0,0 +1,9 @@
+import { helper } from '@ember/component/helper';
+import { set } from '@ember/object';
+
```

Comment:
can we add a comment here explaining why we need this helper?

Reply:
Done